vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -23183,6 +23196,19 @@ unit:M3-PER-HA
   rdfs:label "Cubic Meter per Hectare"@en-us ;
   rdfs:label "Cubic Metre per Hectare"@en ;
 .
+unit:M3-PER-HA-YR
+  rdf:type qudt:Unit ;
+  qudt:conversionMultiplier 0.000000000003168808781402895 ;
+  qudt:conversionMultiplierSN 3.168808781402895e-12 ;
+  qudt:hasDimensionVector qkdv:A0E0L1I0M0H0T-1D0;
+  qudt:plainTextDescription "power of the SI base unit metre with the exponent 3 divided by the unit hectare and by the unit year";
+  qudt:hasQuantityKind quantitykind:VolumeFlowRate_SurfaceRelated ;
```

Comment:
Now you need to create this QuantityKind :)

Reply:
Is SpecificVolumeFlowRate a term of art?

Reply:
Actually, just found that we already have quantitykind:SurfaceRelatedVolumeFlow, so you just need to change the reference.